packages/bson-transpilers/test/yaml/driver-syntax.yaml

```diff
@@ -366,6 +390,28 @@ tests:
                       max_time_ms: max_time_ms,
                       collation: collation
                     })
+                rust: |-
+                    // Requires the MongoDB crate.
+                    // https://crates.io/crates/mongodb
+
+                    let client = Client::with_uri_str("mongodb://localhost:27017").await?;
+                    let options = mongodb::options::FindOptions::builder()
+                        .projection(doc! {
+                      "y": ObjectId::new()
+                    })
+                        .sort(doc! {
+                      "name": 1
+                    })
+                        .skip(5)
+                        .limit(10)
+                        .max_time(std::time::Duration::from_millis(100))
+                        .collation(doc! {
+                      "locale": "zh"
+                    })
+                        .build();
+                    let result = client.database("db").collection("collection").find(doc! {
```

Comment:
we'll probably want to use `Document` as a generic parameter to `collection` here since I don't think the compiler can infer the type otherwise in this example.

Reply:
Good catch, done.